kujira/app/components/test-graph.js

```diff
@@ -0,0 +1,21 @@
+import Ember from 'ember';
+import Mixin from 'kujira/mixins/socketmixin';
+
+export default Ember.Component.extend(Mixin, {
+    dataHandler: function(data) {
+      console.log('zmieniona funkcja dataHandler');
+      console.log(data);
+    },
+
+    onInit: function() {
+      this._super();
+      this.set('graphType', 'speedDataGraph');
```

Comment:
With this lane all components which will extend this Mixin will have to override whole onInit function.
move initalization of graphType to class attributes.